RELEASE_NOTES.md

```diff
@@ -0,0 +1,17 @@
+Release Notes
+=============
+
+v1.0.0-alpha.x
+--------------
+
```

Comment:
[fix] We need a breaking change that the module structure has changed - we didn't have the namespaces before.

Reply:
Added a full listing of the trait name transformations.

[faa24fe](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/15/commits/faa24fec68faec6cd60dc7711ec8412abc719b0d)

Reply:
Nice one 👍 I think we also need to call out the class paths have changed too, maybe as another entry?  Technically the changes listed here are "Trait ID changes" (ie, the string that is actually passed around in the data structure).

Reply:
This the sort of thing you meant? [4f6a70c](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/15/commits/4f6a70cdeeced904362a7ce0ddc0af13dea97b5f)